components/match2/wikis/ageofempires/match_summary.lua

```diff
@@ -0,0 +1,177 @@
+---
+-- @Liquipedia
+-- wiki=ageofempires
+-- page=Module:MatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local DateExt = require('Module:Date/Ext')
+local Game = require('Module:Game')
+local Faction = require('Module:Faction')
+local Icon = require('Module:Icon')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local VodLink = require('Module:VodLink')
+
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper')
+local MatchSummary = Lua.import('Module:MatchSummary/Base')
+
+local OpponentLibraries = require('Module:OpponentLibraries')
+local Opponent = OpponentLibraries.Opponent
+
+local GREEN_CHECK = Icon.makeIcon{iconName = 'winner', color = 'forest-green-text', size = 'initial'}
+local DRAW_LINE = Icon.makeIcon{iconName = 'draw', color = 'bright-sun-text', size = 'initial'}
+local NO_CHECK = '[[File:NoCheck.png|link=]]'
+
+local LINKDATA = {
+	mapdraft = {
+		text = 'Map Draft',
+		icon = 'File:Map Draft Icon.png'
+	},
+	civdraft = {
+		text = 'Civ Draft',
+		icon = 'File:Civ Draft Icon.png'
+	}
+}
+
+local CustomMatchSummary = {}
+
+function CustomMatchSummary.getByMatchId(args)
+	return MatchSummary.defaultGetByMatchId(CustomMatchSummary, args, {width = '285px'})
+end
+
+function CustomMatchSummary.createHeader(match)
+	local header = MatchSummary.Header()
+
+	header:leftOpponent(header:createOpponent(match.opponents[1], 'left'))
+			:leftScore(header:createScore(match.opponents[1]))
+			:rightScore(header:createScore(match.opponents[2]))
+			:rightOpponent(header:createOpponent(match.opponents[2], 'right'))
+
+	return header
```

Comment:
```suggestion
	return MatchSummary.Header()
		:leftOpponent(header:createOpponent(match.opponents[1], 'left'))
		:leftScore(header:createScore(match.opponents[1]))
		:rightScore(header:createScore(match.opponents[2]))
		:rightOpponent(header:createOpponent(match.opponents[2], 'right'))
```

completely optional, but should work too

Reply:
It's accessing functions on `header`, so doesn't work. Can directly return line 48 tho.

Reply:
right, didn't notice that on phobe